modules/gdnavigation/config.py

```diff
@@ -1,5 +1,5 @@
 def can_build(env, platform):
-    return True
+    return not env["disable_3d"]
```

Comment:
Navigation is also used for 2D no?

Reply:
All of this code uses Vector3. Unless 2D navigation uses Vector3, for some reason?

Reply:
I recall reduz and folks suggesting that existing navigation code for 3D can be adapted to work for 2D as well.

Reply:
NavigationServer2D is a wrapper around NavigationServer3D.
I don't know how the servers interface with the `gdnavigation` module exactly but the code already has `_3D_DISABLED` checks for parts of it, and not all of it, which likely means that some parts are needed for `!3D`.